scripts/README.md

```diff
@@ -17,7 +17,34 @@
   * Commit changes and push to origin
   * Open github to submit the PR
 
-## Update sitemap
+### Extracting texts from js code:
+Texts that used in js code would automatically be extracted during the translation update process (3rd step above: Updating the source file).
 
-* List of paths to include in `sitemap.xml` is here: [sitemap.js](sitemap.js)
+In order to make sure that the extractor code won't miss any string to push to translations, please:
+
+1. Refactor the code so that the first argument passed to `localize()` method, be a string literal.
+    i.e.
+    ```js
+    const text = localize(is_started ? 'Sell at market' : 'Sell');
+    ```
+    could change to:
+    ```js
+    const text = is_started ? localize('Sell at market') : localize('Sell');
+    ```
+2. If there is no way to have the string literal in js code (i.e. API texts which are not translated), add them to `scripts/js_texts/static_strings_app.js`.
```

Comment:
should we mention that even if it's a string that's translated automatically somewhere else we should still include it here to avoid missing it if that other string gets removed?

Reply:
@negarn I added this as a [note](https://github.com/binary-com/binary-static/pull/4459/files#diff-efc0b58e058662dd930679e42e8517ecR36). Could you please review again?
Thanks.

Reply:
looks good, thank you